docker-compose.yml

```
@@ -0,0 +1,29 @@
+version: "3.9"
+    
+services:
+  waltz:
+    image: ghcr.io/ljubon/waltz:postgresql 
+    restart: always
+    command: ["update","run"]
+    ports:
+      - "8080:8080"
+    depends_on:
+      - postgres
+    environment:
+      DB_HOST: "postgres"
+      DB_PORT: "5432"
+      DB_NAME: "waltz"
+      DB_USER: "waltz"
+      DB_PASSWORD: "waltz"
+
+  postgres:
+    image: postgres:10.6
```

Use the same version of postgres is used during build

```suggestion
    image: postgres:11
```